examples/config-conformance.json

```diff
@@ -11,5 +11,10 @@
   },
   "log": {
     "level": "debug"
+  },
+  "extensions": {
+    "search": {
+      "enable": true
```

Comment:
Why do we need this change? Search should be optional. Aren't we checking the DB is initialized before attempting to write to it?